apache2/mod_security2.c

```diff
@@ -100,6 +100,7 @@ struct waf_lock DSOLOCAL *wafjsonlog_lock = NULL;
 apr_file_t DSOLOCAL *msc_waf_log_fd = NULL;
 char DSOLOCAL msc_waf_log_path[WAF_LOG_PATH_LENGTH] = ""; 
 cmd_parms DSOLOCAL *msc_waf_log_cmd = NULL;
+char DSOLOCAL *msc_waf_signature = NULL;
```

Comment:
can there be concurrency issues with this global variable? Do we need a mutex? If there are multiple policies that have different CRS versions, what will happen?